### PR TITLE
ENH: consolidate ignore codes

### PR DESCRIPTION
**Pull request by lvjoebro**
dev config changed as well

### CODE REVIEW

File: environment-dev.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ dependencies:
   - geofileops
   - geopandas>=0.11
   - h5py
-  - numpy
+  - numpy<2
```
Comment (lvjoebro) [inline environment-dev.yml:12]: FYI: added this because this gave me a dirty breaking error when it automatically installed numpy 2.0 in my env

Comment (theroggy) [reply environment-dev.yml:12]: Hmm... that's weird. In the ci envs numpy 2.0 is also used, but this doesn't give errors. Did you create a new environment or updated an existing one? If you updated an existing one maybe try to create a new one?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 ## 0.3 (yyyy-mm-dd)
 
+### Deprecations and compatibility notes
+- Consolidated some landcover pre-processing ignore codes
```
Comment (theroggy) [inline CHANGELOG.md:6]: Best add the PR id between brackets...

File: environment-dev.yml (hunk 1 above)
Comment (lvjoebro) [reply environment-dev.yml:12]: see #122

**Pull request merged**